config.yaml

```diff
@@ -118,7 +117,7 @@ collate:
     walltime: 1:00:00
     mpi: false
 
-restart: /g/data/vk83/configurations/inputs/access-esm1p5/modern/pre-industrial/restart/
+restart: /g/data/vk83/prerelease/configurations/inputs/access-esm1p6/modern/pre-industrial/restart/2024.12.10
```

Comment:
does this restart have the latest restart file I sent you? The one from yesterday with ESM hack-a-thon?

Reply:
Yep! It has the `ocean_wombatlite.res.nc-20241211` file, (which is then renamed as `ocean_wombatlite.res.nc-01001231-v20241211`)